components/dashboard/src/components/BillingAccountSelector.tsx

```diff
@@ -96,7 +96,17 @@ export function BillingAccountSelector(props: { onSelected?: () => void }) {
             {teamsAvailableForAttribution === undefined && <Spinner className="m-2 h-5 w-5 animate-spin" />}
             {teamsAvailableForAttribution && (
                 <div>
-                    <p>Associate usage without a project to the billing account below.</p>
+                    <p>
+                        Associate usage without a project to the billing account below.{" "}
+                        <a
+                            className="gp-link"
+                            href="https://www.gitpod.io/docs/configure/billing"
+                            target="_blank"
+                            rel="noreferrer"
+                        >
+                            Learn more
+                        </a>
+                    </p>
```

Comment:
**Thought (no action required):** 

* **Pro:** Placing links here means that we can have a "learn more" for each section on a settings page.
* **Con:** Placing here means that 

**Suggestion (non-blocking):** We could elevate to the header area, and potentially use an information icon, as the link blue is quite eye-catching (and I don't think we need it to be super eye-catching. If a user is stuck, they'll actively seek out documentation or help links anyway). A question for @gtsiolis, really. 

<img width="863" alt="image" src="https://user-images.githubusercontent.com/5528307/208837539-a4cc4112-154f-4abc-8b7a-5646ffd9dfdd.png">

Reply:
> We could elevate to the header area, and potentially use an information icon

Agreed ✅. Yeah, George would be the perfect one to answer this 🧡